shell/browser/ui/inspectable_web_contents.cc

```diff
@@ -918,6 +918,36 @@ void InspectableWebContents::GetPreferences(DispatchCallback callback) {
   std::move(callback).Run(&settings);
 }
 
+void InspectableWebContents::GetPreference(DispatchCallback callback,
```

Comment:
Was this addition required by a change in Chromium? If so, would be great to have a link to the CL!

Reply:
cc @clavin do you recall which CL this was added in?

Reply:
I had asked @clavin about this while I was working on the 17-x-y roll and he said it was due to this CL:
3302814: DevTools: Add getPreference binding | https://chromium-review.googlesource.com/c/chromium/src/+/3302814